scripts/check_regions.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+AWS_REGIONS=$(aws ec2 describe-regions | jq -r '.[] | .[] | .RegionName')
+LAYER_NAMES=("Datadog-Node10-x" "Datadog-Node12-x")
+FOUND_REGIONS_MISSING_LAYERS=false
+
+for layer_name in "${LAYER_NAMES[@]}"; do
+    for region in $AWS_REGIONS; do
+        last_layer_arn=$(aws lambda list-layer-versions --layer-name $layer_name --region $region | jq -r ".LayerVersions | .[0] |  .LayerVersionArn")
+        if [ "$last_layer_arn" == "null" ]; then
+             echo "No layer found for $region, $layer_name"
+             FOUND_REGIONS_MISSING_LAYERS=true
+        fi
+    done
+done
+
+if [ $FOUND_REGIONS_MISSING_LAYERS = true ]; then
+    echo "Run ./add_new_region.sh <new_region> to add layers to the missing regions" 
+fi
```

Comment:
It looks like this prints warnings if we have missing regions -- is that going to be conspicuous enough to get our attention when we publish the layer?

We could put `WARNING:` at the beginning of the message but I wonder if people still might miss it...

Reply:
I like Darcy's idea of actually failing here, if that is possible

Reply:
✅